layouts/index.html

```diff
@@ -246,12 +243,17 @@ <h5>Blockchains</h5>
             </div>
             <h5>Content creators</h5>
             <p>
-              IPFS brings the freedom and independent spirit of the web at full
-              force &mdash; and it can help you deliver content while saving
-              considerable cost.
+              IPFS brings the freedom and independent spirit of the web in full
+              force &mdash; and can help you deliver your content at a lower cost.
```

Comment:
Traded `considerable` for `lower` which doesn't feel strong enough... maybe `much lower`?

(And I *really* want to have some cost projections for different scenarios at some point to back this up!)

Reply:
"much lower" is great, thanks!